timer/runtime/runtime.go

```diff
@@ -145,12 +150,16 @@ func (rt *TimerGroupRuntime) Stop() {
 	rt.wg.Wait()
 }
 
-func (rt *TimerGroupRuntime) loop() {
-	rt.logger.Info("TimerGroupRuntime loop started")
-	defer func() {
-		rt.logger.Info("TimerGroupRuntime loop exit")
-		rt.wg.Done()
-	}()
+func (rt *TimerGroupRuntime) loop(totalPanic uint64) {
+	if totalPanic > 0 {
+		sleep(rt.ctx, rt.retryLoopWait)
+		rt.logger.Info("TimerGroupRuntime loop resumed from panic",
```

Comment:
I think using `Warn` is better? Ditto for the rest of `panic` logs.

Reply:
The panic have already been logged in `WithRecover` function, here is just to information that the loop resumed again